lychee-bin/src/main.rs

```diff
@@ -314,7 +314,7 @@ async fn run(opts: &LycheeOptions) -> Result<i32> {
         collector
     };
 
-    let requests = collector.collect_links(inputs);
+    let requests = collector.collect_links_with_ext(inputs, opts.config.extensions.clone());
```

Comment:
I'd actually prefer `collect_links` over `collect_links_with_ext` because the latter sounds like the link itself is checked for some extension.

Reply:
Ah, good point. I can switch it around. So what should the method that takes a list of extensions be called?